### PR TITLE
Dcat issue 1364 second solution

### LINKED ISSUE #1153: Inconsistency between usage note of dcat:themeTaxonomy and range of dcat:theme?
The usage note of dcat:themeTaxonomy says "It is recommended that the taxonomy is organized in a skos:ConceptScheme, skos:Collection, owl:Ontology or similar, which allows each member to be denoted by an IRI and published as Linked Data.". 
But the range of dcat:theme is skos:Concept, so it seems that the expectation for the members of the objects of dcat:themeTaxonomy statements is a bit more precise than what the usage note says.

Future work?

### LINKED ISSUE #1364: Relax range restriction on dcat:theme
@dr-shorthair 

Why not relax range constraint for `dcat:theme`? In the biomedical domain we have more ontologies and often these ontologies have `owl:Class` typing for concepts.

_Originally posted by @rajaram5 in https://github.com/w3c/dxwg/issues/123#issuecomment-843290546_

### PR DESCRIPTION
**Pull request by riccardoAlbertoni**
An alternative solution to PR #1380 

This  PR drops the range for `dcat:theme`, but explicitly defining it as `owl:ObjectProperty` as suggested by @dr-shorthair  in https://github.com/w3c/dxwg/issues/1364#issuecomment-843616224

Considering that `dcterm:subject` has been recently changed to let both literals and  URI,  by defining `dcat:theme`s as an `owl:ObjectProperty`  and dropping the `dcat:theme`'s range,  `dcat:theme` would still restrict `dcterms:subject` to URIs,  while `dcat:keyword` might restrict `dcterms:subject` to literals.

This would also limit the automatic inference of `skos:Concept`, which could be potentially problematic when applied to instances whose classes are defined disjoint from `skos:Concept`.

### CODE REVIEW

**Comment by dr-shorthair** [comment]
This looks clean and sensible to me

**Comment by riccardoAlbertoni** [comment]
@andrea-perego  can you check the rephasing of the note before defining`dcat:theme` meets the expectation you have expressed in the last DCAT teleconference?  Feel free to rephrase it, otherwise. Thanks.

File: dcat/index.html [hunk 1]
```diff
@@ -1258,15 +1258,17 @@ <h4>Property: theme/category</h4>
...
                 See Issue <a href="https://github.com/w3c/dxwg/issues/123">#123</a>.
+            </p>
+            <p> 
+                DCAT 3 maintains the same expectation as DCAT 2 for the use of the property <code>dcat:theme</code>, . DCAT 3 defines <code>dcat:theme</code> as an OWL object property and  drops its range to  make the formalization of the property more consistent with the expected use. In particular, the change intends to prevent that each entity used as object of <code>dcat:theme</code> is automatically inferred as <code>skos:Concept</code>. See issue <a href="https://github.com/w3c/dxwg/issues/1364">#1364</a>.
               </p>
             </aside>
 
             <table class="def">
-                <tr><th>RDF Property:</th><td><a href="http://www.w3.org/ns/dcat#theme"><code>dcat:theme</code></a></td></tr>
+                <tr><th>OWL Object Property:</th><td><a href="http://www.w3.org/ns/dcat#theme"><code>dcat:theme</code></a></td></tr>
```
Comment (andrea-perego) [inline dcat/index.html]:
```suggestion
                <tr><th>RDF Property:</th><td><a href="http://www.w3.org/ns/dcat#theme"><code>dcat:theme</code></a></td></tr>
                <tr><th>Type:</th><td><a href="http://www.w3.org/2002/07/owl#ObjectProperty"><code title="http://www.w3.org/2002/07/owl#ObjectProperty">owl:ObjectProperty</code></a></td></tr>```

**Review by andrea-perego — CHANGES_REQUESTED** [review]
Thanks, @riccardoAlbertoni .

+1 from me. I just suggested an editorial change.

**Comment by riccardoAlbertoni** [comment]
> Thanks, @riccardoAlbertoni .
> 
> +1 from me. I just suggested an editorial change.

Thanks, Andrea. I have accepted your suggestion.

**Review by andrea-perego — APPROVED** [review]

**Comment by riccardoAlbertoni** [comment]
@davebrowning , @pwin , @agbeltran , Is this PR ok for you?

**Review by davebrowning — APPROVED** [review]

**Pull request merged**